applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -433,6 +437,33 @@ export const RepositoryPage = () => {
     });
   }
 
+  const getImages = (str: string) => {
+    const imgRex = /<img.*?src="(.*?)"[^>]+>/g;
+    const imageTags = [];
+    const imagePaths = [];
+    let img;
+    while ((img = imgRex.exec(str))) {
+      imageTags.push(img[0]);
+      imagePaths.push(img[1]);
+
+    }
+    return [imageTags, imagePaths];
+  }
+
+  const convertImgInMarkdown = (markDown: string, repositoryUri: string) => {
+    let mark = markDown;
+    const [imageTags, imagePaths] = getImages(markDown);
+    const updatedImages : string[] = [];
+    imagePaths.map((image) => {
+      const y = `[![img](${repositoryUri.replace('https://github.com/', 'https://raw.githubusercontent.com/')}/master/${image})](${repositoryUri.replace('https://github.com/', 'https://raw.githubusercontent.com/')}/master/${image})?target=_blank`;
```

Comment:
must take repository uri and context ( branch may be different from master). Also, images may come from outside the repo (can be any in a markdown). Need to think about this